examples/analytics/dist/index.html

```diff
@@ -1,9 +1,8 @@
-<!doctype html><html><head><meta charset="utf-8"><script async src="https://www.googletagmanager.com/gtag/js?id=UA-28076707-7"></script><script>window.dataLayer = window.dataLayer || [];
+<!doctype html><html><head><script async="" src="https://www.googletagmanager.com/gtag/js?id=UA-28076707-7"></script><script>window.dataLayer = window.dataLayer || [];
 function gtag(){dataLayer.push(arguments);}
 gtag('js', new Date());
 
-gtag('config', 'UA-28076707-7');</script><title>Webpack App</title><meta name="viewport" content="width=device-width,initial-scale=1"></head><body><h1>Hello world!</h1><hr/><h2>webpack.config.js</h2><pre>
-const path = require('path');
+gtag('config', 'UA-28076707-7');</script><meta charset="utf-8"><title>Webpack App</title><meta name="viewport" content="width=device-width,initial-scale=1"></head><body><h1>Hello world!</h1><hr><h2>webpack.config.js</h2><pre>const path = require('path');
```

Comment:
oh looks like it was just stuffed behind the script tag

this is what i was referring to in the comments from https://github.com/colbyfayock/html-webpack-partials-plugin/issues/31

i think the meta tag needs to go before any content we inject or we risk issues with how that content loads